3.7/programs-starter-architecture.md

```diff
@@ -93,6 +93,109 @@ It contains the following files & sub-directories.
   - `arangod.log`: The log file of the server
   - `arangod_command.txt`: File containing the exact command line of the started server (for debugging purposes only)
 
+## Starter configuration file
+
+The Starter can be configured using a configuration file. The format of the
+configuration file is the same as the `arangod` configuration file format.
+For more details, refer to the [General Configuration of ArangoDB programs and tools](administration-configuration.html#configuration-file-format).
+
+The default configuration file of the Starter is `arangodb-starter.conf`.
+It can be changed using the `--configuration` option. 
+For more information about supported commands and other
+configuration options, see [Using Configuration Files](administration-configuration.html#using-configuration-files). 
+
+{% hint 'info' %}
+The Starter has a different set of supported command line options.
+Using the `arangod` configuration file as input for `arangodb` binary is not supported.
+{% endhint %}
+
+### Basic example
+
+You can run the Starter without specifying any command line options or only
+by specifying the path of the configuration file. 
+
+```conf
+# config file '/etc/arangodb-starter.conf'
+
+[starter]
+local = true
+data-dir = /var/local/arangodb
+```
+
+```conf
+./arangodb --configuration=/etc/arangodb-starter.conf
+```
+
+### Invalid file path
+
+In case of an invalid file path, the Starter returns an error message. For
+example, specifying a configuration file that does not exist returns 
+the following output:
+
+```bash
+➜ ./arangodb --configuration=invalid-config-file.conf
+2022-08-22T12:08:10+02:00 |FATAL| Could not load config file 
+invalid-config-file.conf
+➜ echo $?                                    
+1
+```
+
+### Passing through options
+
+The configuration file also supports setting pass through options. Options with
+same prefixes can be split into sections.
+
+```conf
+# passthrough-example.conf
+
+# both log.level options are passed to arangod, with no overrides
```

Comment:
```suggestion
```
I think this comment can be removed too, because it duplicates info from `ArangoD` configuration manual